examples/with-multiple-theme-providers/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "with-multiple-theme-providers",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "@types/jest": "^27.5.2",
+    "@types/node": "^16.18.3",
+    "@types/react": "^18.0.25",
+    "@types/react-dom": "^18.0.9",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-scripts": "5.0.1",
+    "typescript": "^4.9.3",
+    "web-vitals": "^2.1.4",
+    "@razorpay/blade": "../../packages/blade/build/",
+    "@fontsource/lato": "^4.5.10",
+    "styled-components": "^5.3.6",
+    "@types/styled-components": "5.1.25"
```

Comment:
nit: use exact package versions

Reply:
Done in https://github.com/razorpay/blade/pull/889/commits/ed10d115f7b9898ba8e3a5cb236ef463a1496cb5